src/sql/workbench/contrib/dashboard/browser/core/dashboardPage.css

```diff
@@ -60,5 +60,8 @@ dashboard-page .editor-toolbar {
 	align-items: center;
 	border-bottom-width: 1px;
 	border-bottom-style: solid;
-	height:26px;
+}
+
+dashboard-page .editor-toolbar .monaco-action-bar {
+	padding: 5px 0;
```

Comment:
why isn't this with the other toolbar css above?

Reply:
:) updated, moved this to the css class above, and use padding 3px instead of 5px, before 5px was used to replace the builtin 2px